### PR TITLE
add .packit.yaml

### PR DESCRIPTION
**Pull request by jpopelka**
I couldn't help myself, I can't live with json as a config file.

### CODE REVIEW

**Comment by TomasTomecek** [comment]
jeeesh, now I need to update my packit PR since I'm referencing the json config -_- :P :D :dango:

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+specfile_path: colin.spec
+synced_files:
+  - .packit.json
```
Comment (rpitonak) [inline .packit.yaml]: Did you mean `.packit.yaml`?

Comment (TomasTomecek) [reply .packit.yaml]: and why do we need to sync it?

Comment (lachmanfrantisek) [reply .packit.yaml]: So you can more easily run packit from distgit? (This would need some work on the packit site to make it usable.)

Comment (jpopelka) [reply .packit.yaml]: good catch

Comment (TomasTomecek) [reply .packit.yaml]: but since packit can't do it yet, we shouldn't add it; I actually have one more suggestion, will create a PR instead

**Pull request merged**